frontend/app/tests/unit/components/display/amount-display.spec.ts

```diff
@@ -33,19 +35,14 @@ describe('AmountDisplay.vue', () => {
   let wrapper: Wrapper<any>;
   let pinia: Pinia;
 
-  const createWrapper = (
-    value: BigNumber,
-    amount: BigNumber,
-    fiatCurrency: string | null
-  ) => {
+  const createWrapper = (value: BigNumber, props: Object = {}) => {
```

Comment:
Do we use that many properties here? maybe we could type it properly for code assistance.